pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -2224,6 +2322,37 @@ func (c *Cloud) DisksAreAttached(nodeDisks map[types.NodeName][]KubernetesVolume
 	return attached, nil
 }
 
+func (c *Cloud) ResizeDisk(
+	diskName KubernetesVolumeID,
+	oldSize resource.Quantity,
+	newSize resource.Quantity) (resource.Quantity, error) {
+	awsDisk, err := newAWSDisk(c, diskName)
+	if err != nil {
+		return oldSize, err
+	}
+
+	volumeInfo, err := awsDisk.describeVolume()
+	if err != nil {
+		descErr := fmt.Errorf("AWS.ResizeDisk Error describing volume %s with %v", diskName, err)
+		return oldSize, descErr
```

Comment:
Any reason to bother creating descErr, vs `return oldSize, fmt.Errorf...`

Reply:
No strong reason. I just don't like it when certain code line becomes too long and starts to become wrapped in the code editor.